week1/main.js

```diff
@@ -0,0 +1,65 @@
+// This is a class based component that has two props that are reuseable
+// as demonstrated in lines 47 through 49.
+class ListItems extends React.Component {
+  render() {
+    const { description, deadline } = this.props;
+    return (
+      <div>
+        {description}, {deadline}
+      </div>
+    );
+  }
+}
+
+// This component serves as parent component which renders the component
+// being nested in it.
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          id: 1,
+          description: 'Get out of bed',
+          deadline: '2017-09-13',
+          done: true,
+        },
+        {
+          id: 2,
+          description: 'Brush teeth',
+          deadline: '2017-09-14',
+          done: false,
+        },
+        {
+          id: 3,
+          description: 'Eat Breakfast',
+          deadline: '2017-09-15',
+          done: false,
+        },
+      ],
+    };
+  }
+  render() {
+    return (
+      <div className="list-items">
+        <h1 className="title">Static List</h1>
+        <ListItems description="Get out of bed" deadline="2017-09-13" />
```

Comment:
Please make a container component which holds the static list, and the same for the dynamic list. Two components called `StaticList` and `DynamicList` for example. Pass the right data to these container components and renders the `ListItem`s.